core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -48,25 +49,12 @@
   }
 
   private static void formatFolder(String name, String folder) throws IOException {
-    UnderFileSystem ufs = UnderFileSystem.Factory.get(folder);
```

Comment:
To follow up on this, previously `formatFolder` was used for the journal folders which could be on under storage systems, but that logic has been moved, so now it is only used when formatting the ramdisk.